### PR TITLE
Fix logPrefix path

### PR DESCRIPTION
**Pull request by irisdianauy**
When work directory `-w` and log prefix `-l` are both supplied, the work directory is automatically made base directory of the log prefix, and this creates file not found error when an absolute path is supplied for the log.  This is a fix for such errors.

### CODE REVIEW

File: atlas-slurm [hunk 1]
```diff
@@ -128,7 +128,7 @@ else
             logFile=${logPrefix}.out
         fi 
 	sleep 10
-	slurm_monitor_job "$slurmJobId" "$pollFreqSecs" "${workingDir}/${logFile}" "$monitorStyle" "$logCleanup" "$returnStdout" "$quiet"
+	slurm_monitor_job "$slurmJobId" "$pollFreqSecs" "$logFile" "$monitorStyle" "$logCleanup" "$returnStdout" "$quiet"
```
Comment (anilthanki) [inline atlas-slurm:131]: have you tested with this? It may fail or throw error

Comment (anilthanki) [reply atlas-slurm:131]: especially when just file name is specified not path

Comment (irisdianauy) [reply atlas-slurm:131]: So the thing is, in `slurm_monitor_job` the basename of `logFile` should be consistent with the `logPrefix` given in `slurm_submit` [here](https://github.com/ebi-gene-expression-group/atlas-bash-util/blob/2102391f8f986f73bb85b127a76f83adf1f4e760/atlas-slurm#L117).  The change above makes them consistent.  The log is written in the supplied log prefix (no filename is supplied by the user, only a prefix).  If there is no path in the prefix (only a basename), the log is written in the present working dir (not the supplied `workingDir`), as is the behavior in `atlas-lsf`.

Comment (irisdianauy) [reply atlas-slurm:131]: Forgot to also say that yes this has been tested and is working where only logprefix is only a file prefix (no path). Log is written on the same dir the job is submitted.

**Review by anilthanki — APPROVED** [review]

**Review by pmb59 — APPROVED** [review]
LGTM @anilthanki !

**Pull request merged**